Plugins/Public/base_plugin/CreateSolar.cpp

```diff
@@ -1,17 +1,34 @@
 #include "Main.h"
 
+void CreateSolar::DespawnSolarCallout(DESPAWN_SOLAR_STRUCT* info)
+{
+	if (!customSolarList.count(info->spaceObjId))
+	{
+		return;
+	}
+
+	if (pub::SpaceObj::ExistsAndAlive(info->spaceObjId) == 0)
+	{
+		pub::SpaceObj::Destroy(info->spaceObjId, info->destroyType);
```

Comment:
We destroy if it either doesn't exist or is not alive?
Wonder if the return codes of ExistsAndAlive should be better shown here, or if this is just wrong

Reply:
0 means alive, -2 means dead, no other values are returned. I've added this exact note to several other places this method is called, but I guess I missed this one.